core-bundle/src/Fragment/Reference/ContentElementReference.php

```diff
@@ -30,4 +30,12 @@ public function __construct(ContentModel $model, string $section = 'main', array
         $this->attributes['classes'] = $model->classes;
         $this->attributes['templateProperties'] = $templateProperties;
     }
+
+    public static function withModel(ContentModel $model, string $section = 'main', array $templateProperties = []): ContentElementReference
```

Comment:
I don't like the naming because the constructor also accepts a model. How about `createForInlineRenderer()` or so?